File: html/old/backup/sacor2_2008/heron.java
// SACO Second Round 2008
// Problem: Heron's Forumla
// Author:  Marco Gallotta

import java.io.*;
import java.util.Scanner;
public class herons {
  public static void main(String args[]) throws IOException {
    Scanner in = new Scanner(System.in);
    System.out.print("Enter a: ");
    int a = in.nextInt();
    System.out.print("Enter b: ");
    int b = in.nextInt();
    System.out.print("Enter c: ");
    int c = in.nextInt();
    double s = (a+b+c)/2.0;
    double area = Math.sqrt(s * (s-a) * (s-b) * (s-c));
    System.out.println(area);
  }
}

File: html/old/backup/sacor2_2008/heron.py
#!/usr/bin/env python
# SACO 2nd Round 2008
# Solution by Richard Baxter
import math 
a = int(raw_input("Enter a: ")) 
b = int(raw_input("Enter b: "))
c = int(raw_input("Enter c: "))
s = 1.0*(a+b+c)/2
print math.sqrt(s*(s-a)*(s-b)*(s-c))
